test/health-score-test.js

```diff
@@ -1,11 +1,97 @@
 const { assert } = require('chai');
-// const sinon = require('sinon');
-// const core = require('@actions/core');
-// const github = require('@actions/github');
+const sinon = require('sinon');
+const { fakeCore, fakeGithub } = require('./stubs/stubs');
 const hs = require('../src/health-score');
 
+const contextValue = {
+  eventName: 'pull_request',
+  payload: {
+    after: 'abcd1234',
+  },
+  repo: {
+    owner: 'slackapi',
+    repo: 'slack-health-score',
+  },
+};
+
 describe('health-score', () => {
+  const fakeComments = sinon.stub(hs, 'grep');
+  const fakeCoverage = sinon.stub(hs, 'coverage');
+  const originalGithubContext = fakeGithub.context;
+  beforeEach(() => {
+    sinon.reset();
+    fakeGithub.context = originalGithubContext;
+  });
   it('should have a grep function', async () => {
     assert.ok(hs.grep);
   });
+  describe('check: inputs', () => {
+    it('should handle invalid input', async () => {
```

Comment:
Good question raised by this test: how should invalid input be handled? I am leaning towards erroring out and reporting the invalidity of inputs using `core.error`. What do you think?

Reply:
I was infact checking for blank or null inputs by invalid inputs. Yes i guess we can go for a `core.error` reporting. I was initially thinking of defaulting to the entire repo, but yes that would be kind of pointless, especially with not gitignored `node_modules` repos. Will add a `core.error` invocation incase of invalid inputs.